TowerForge/TowerForge/GameModule/Events/GameEvents/DamageEvent.swift

```diff
@@ -9,21 +9,31 @@ import Foundation
 
 // Should generate this on collision between an entity with a damage component and a health component
 struct DamageEvent: TFEvent {
+    var isActive: Bool
```

Comment:
This won't be needed if an nil event is returned at the transform.

Reply:
I added a `DisabledEvent` which I think will avoid the optional value but follows the same idea.